tests/step-110.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+# Test:
+#   Trigger hooks on a bare repo with a push from a local repo.
+
+if ! curl --version && ! wget --version; then
+    echo "Neither curl nor wget is available"
+    exit 249
+fi
+
+if ! sh /var/lib/githooks/install.sh; then
+    echo "! Failed to execute the install script"
+    exit 1
+fi
+
+mkdir -p /tmp/test110/p001 &&
```

Comment:
Can you rename the `p001`, `p002` and `p003` directories to something more meaningful?
This test ended up somewhat long and it's quite hard to follow what is what.

Reply:
Clear :-)